update.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+echo "updating sjtu-plus-docker..."
+(cd ./ && git fetch && git reset --hard origin/master)
```

Comment:
I suggest not resetting here, as there might be local changes.

Reply:
Should there be local changes in production environment?